examples/hotrod/README.md

```diff
@@ -28,6 +28,17 @@ to view the traces. A tutorial / walkthrough is available:
 
 Alternatively, you can run each component separately as described below.
 
+### Run HotROD/Jaeger all-in-one from kubernetes
+
+```bash
+kustomize build ./kubernetes | kubectl apply -f -
+kubectl port-forward -n example-hotrod service/example-hotrod 8080:frontend
+# In another terminal
+kubectl port-forward -n example-hotrod service/jaeger 16686:frontend
+```
+
+Then open <http://127.0.0.1:8080>
+
```

Comment:
how to clean up?

Reply:
Added https://github.com/jaegertracing/jaeger/pull/3645/commits/5477e7e5cfeabe581042100ef0c230bbebc0248a